runtime/Pervasives_mini.res

```diff
@@ -1,5 +1,6 @@
 /* Exceptions */
 external raise: exn => 'a = "%raise"
```

Comment:
should we deprecate raise or keep both for now?

Reply:
I will remove